### PR TITLE
Remove old vortex dependency

### PR DESCRIPTION
**Pull request by yudilevi**

### CODE REVIEW

File: deploy/scripts/mainnet/0017-CarbonVortex-upgrade.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { DeployFunction } from 'hardhat-deploy/types';
+import { HardhatRuntimeEnvironment } from 'hardhat/types';
+import { DeployedContracts, upgradeProxy, InstanceName, setDeploymentMetadata } from '../../../utils/Deploy';
+import { NATIVE_TOKEN_ADDRESS } from '../../../utils/Constants';
+
+/**
+ * upgrade carbon vortex 2.0 to v3:
+ * add maxInput to trade function
```
Comment (ivanzhelyazkov) [inline deploy/scripts/mainnet/0017-CarbonVortex-upgrade.ts]: description here can be 
* remove old vortex dependency

File: contracts/vortex/CarbonVortex.sol [hunk 2]
```diff
@@ -379,7 +373,6 @@ contract CarbonVortex is ICarbonVortex, Upgradeable, ReentrancyGuardUpgradeable,
         // cache address checks to save gas
         bool carbonControllerIsNotZero = address(_carbonController) != address(0);
         bool vaultIsNotZero = address(_vault) != address(0);
-        bool oldVortexIsNotZero = address(_oldVortex) != address(0);
 
         // withdraw fees from carbon, vault and old vortex
```
Comment (ivanzhelyazkov) [inline contracts/vortex/CarbonVortex.sol]: old vortex part of the comment can be removed

**Review by ivanzhelyazkov — APPROVED** [review]

File: deploy/scripts/mainnet/0017-CarbonVortex-upgrade.ts (hunk 1 above)
Comment (yudilevi) [reply deploy/scripts/mainnet/0017-CarbonVortex-upgrade.ts]: done

File: contracts/vortex/CarbonVortex.sol (hunk 2 above)
Comment (yudilevi) [reply contracts/vortex/CarbonVortex.sol]: done

**Review by ivanzhelyazkov — APPROVED** [review]

**Review by lbeder — APPROVED** [review]

**Pull request merged**